examples/basic/android/build.gradle

```diff
@@ -15,6 +24,12 @@ buildscript {
 allprojects {
     repositories {
         mavenLocal()
+        maven {
```

Comment:
could this be `google()`?

Reply:
sorry, my mistake. this could be `google()`